aws/table_aws_backup_job.go

```diff
@@ -0,0 +1,242 @@
+				Description: "The name of the target backup vault.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("BackupVaultName"),
+			},
+			{
+				Name:        "bytes_transferred",
+				Description: "The size in bytes transferred to a backup vault at the time that the job status was queried.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("BytesTransferred"),
+			},
+			{
+				Name:        "completion_date",
+				Description: "The date and time a backup job is completed.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("CompletionDate"),
+			},
+			{
+				Name:        "creation_date",
+				Description: "The date and time a backup job is created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("CreationDate"),
+			},
+			{
+				Name:        "expected_completion_date",
+				Description: "The date and time a backup job is expected to be completed.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("ExpectedCompletionDate"),
+			},
+			{
+				Name:        "iam_role_arn",
+				Description: "The ARN of the IAM role that AWS Backup uses to create the target recovery point.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("IamRoleArn"),
+			},
+			{
+				Name:        "is_parent",
+				Description: "A Boolean value that is returned as TRUE if the specified job is a parent job.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("IsParentJob"),
+			},
+			{
+				Name:        "parent_job_id",
+				Description: "The ID of the parent backup job, if there is one.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ParentJobId"),
+			},
+			{
+				Name:        "percent_done",
+				Description: "The percentage of job completion.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("PercentDone"),
+			},
+			{
+				Name:        "start_by",
+				Description: "The date and time a backup job must be started before it is canceled.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("StartBy"),
+			},
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("BackupJobId"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ResourceArn").Transform(arnToAkas),
+			},
+		}),
+	}
+}
+
+func listAwsBackupJobs(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := BackupClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_backup_vault.listAwsBackupJobs", "connection_error", err)
```

Comment:
```suggestion
		plugin.Logger(ctx).Error("aws_backup_job.listAwsBackupJobs", "connection_error", err)
```
Please address this wherever applicable.